### DISCUSSION TITLE
Any plans to release the ndk for linux but built for arm64?

### DISCUSSION THREAD

**Discussion by avetiso** [android/ndk#1692] [Q&A] [answered]
Our team is interested in using AWS Graviton, which uses ARM cores for cost/performance. We would deploy our tools in a linux container, but there is no arm linux ndk to us.

Are there any plans to release one?

**Comment by enh-google** [comment, marked as answer]
although strictly speaking "linux/arm64 NDK" is somewhat orthogonal to the larger question of "the whole Studio collection", you can see discussion of the wider problem at https://issuetracker.google.com/227219818 which touches on the NDK.

TL;DR is probably something like "no current plans, but we are [for other reasons] working on some of the prerequisites, and the CI use case is something we're thinking about". but short-term, no, linux/x86-64 is the supported CI option.

**Reply by avetiso** [reply]
Thanks for the response!

**Comment by Zingam** [comment]
Microsoft will be delivering Visual Studio 22 for arm on Windows. So I'd expect that in a new too distant future all tools to be supported on arm platforms.
https://www.youtube.com/watch?v=ZEnAyTi1uIQ
